src/pybind/mgr/nfs/cluster.py

```diff
@@ -121,7 +121,7 @@ def create_nfs_cluster(
 
             if cluster_id not in available_clusters(self.mgr):
                 self._call_orch_apply_nfs(cluster_id, placement, virtual_ip, port)
-                return 0, "NFS Cluster Created Successfully", ""
+                return 0, "Creating NFS cluster", ""
```

Comment:
I'm approve of this change, but in general I would encourage traditional unixy practices of not printing stuff out (unless asked) on success. I had to work around the fact that the nfs module does print these "friendly" messages in go-ceph (see: https://github.com/ceph/go-ceph/blob/master/common/admin/nfs/export.go#L141 for an example).
And if we must print stuff out we should be consistent and print structured responses that contain an id/key that could be fed back into a status command.

Reply:
Good points. Out of curiosity, I checked the output of `fs volume create/rm` commands that creates/deletes a CephFS file system and creates/removes MDS daemons asynchronously.

`fs volume create` prints nothing on success
`fs volume rm` prints a message "metadata pool: cephfs.a.meta data pool: ['cephfs.a.data'] removed" on success

We're not consistent with output messages in `fs volume create/rm` commands as well.

I like the idea of not printing anything when the commands `nfs cluster create/rm` succeed. What do you think?

Reply:
> Let me add a mgr/nfs tracker ticket to help keep track of backports.

Done.